src/commands/server/delete.ts

```diff
@@ -39,14 +39,16 @@ export default class Delete extends Command {
     'listr-renderer': listrRenderer,
     'skip-deletion-check': boolean({
       description: 'Skip user confirmation on deletion check',
-      default: false
+      default: false,
+      hidden: true
     }),
-    'skip-kubernetes-health-check': skipKubeHealthzCheck
+    'skip-kubernetes-health-check': skipKubeHealthzCheck,
+    yes: assumeYes
```

Comment:
`confirmation`?

Reply:
assume is more general then confirmation